test/libsolidity/semanticTests/constants/define_constants_with_external_constant_member_var.sol

```diff
@@ -0,0 +1,25 @@
+==== Source: external.sol ====
```

Comment:
This should be in `syntaxTests/`. The returned values are not relevant to us and in a semantic test you don't see the returned error messages.

Reply:
Done